### PR TITLE
Added URP deploy script and docker file and removed plugin-redfish binary 

### PR DESCRIPTION
**Pull request by amar-shalgar**

### CODE REVIEW

File: install/Docker/dockerfiles/scripts/start_urplugin.sh [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	nohup ./plugin-unmanaged-racks >> /var/log/urplugin_logs/urplugin.log 2>&1 &
+	PID=$!
+	sleep 2s
+  nohup /bin/add-hosts -file /tmp/host.append >> /var/log/urplugin_logs/add-hosts.log 2>&1 &
+}
+
+monitor_process()
+{
+        while true; do
+                pid=$(pgrep plugin-unmanage 2> /dev/null)
```
Comment (bharath-b-hpe) [inline install/Docker/dockerfiles/scripts/start_urplugin.sh]: Please include the latest changes made in other start scripts, to monitor the service and restart when it has exited.

**Review by bharath-b-hpe — COMMENTED** [review]
Changes are required in https://github.com/ODIM-Project/ODIM/blob/development/install/Docker/dockerfiles/scripts/build_odim.sh too

**Review by rahulroshan-kachchap — APPROVED** [review]

File: install/Docker/dockerfiles/scripts/start_urplugin.sh (hunk 1 above)
Comment (amar-shalgar) [reply install/Docker/dockerfiles/scripts/start_urplugin.sh]: Addressed the comment

File: install/Docker/dockerfiles/scripts/start_urplugin.sh [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+start_urplugin()
+{
+	export PLUGIN_CONFIG_FILE_PATH=/etc/urplugin_config/config.yaml
+	nohup /bin/plugin-unmanaged-racks >> /var/log/urplugin_logs/urplugin.log 2>&1 &
+	PID=$!
+	sleep 3
+
+	nohup /bin/add-hosts -file /tmp/host.append >> /var/log/iloplugin_logs/add-hosts.log 2>&1 &
```
Comment (irappa-pattar) [inline install/Docker/dockerfiles/scripts/start_urplugin.sh]: `/var/log/iloplugin_logs/add-hosts.log` this log should be in `urplugin_logs`

**Review by irappa-pattar — CHANGES_REQUESTED** [review]

File: install/Docker/dockerfiles/scripts/start_urplugin.sh (hunk 2 above)
Comment (amar-shalgar) [reply install/Docker/dockerfiles/scripts/start_urplugin.sh]: Corrected the path

**Review by irappa-pattar — APPROVED** [review]

**Review by jeevan-kamkar — APPROVED** [review]

**Review by bharath-b-hpe — APPROVED** [review]

**Pull request merged**